src/Jobs/Recruit.Vacancies.Jobs/Jobs/TransferVacanciesFromProviderJob.cs

```diff
@@ -1,27 +1,36 @@
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Esfa.Recruit.Vacancies.Client.Application.Queues;
 using Esfa.Recruit.Vacancies.Client.Application.Queues.Messages;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore;
 
 namespace Esfa.Recruit.Vacancies.Jobs
 {
```

Comment:
namespace can be corrected to point to the right collection.

Reply:
Do you mean change to `namespace Esfa.Recruit.Vacancies.Jobs.Jobs`?

Reply:
yes